### PR TITLE
allow admin to opt out of CLR

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

Untitled

This is duplicate of https://github.com/gitcoinco/web/pulls and uses `is_clr_elgible` instead of adding a new field cause we had a few support requests asking us to remove their grant from CLR

### CODE REVIEW

**Review by chibie — APPROVED** [review]

File: app/grants/views.py [hunk 1]
```diff
@@ -1593,6 +1593,9 @@ def grant_edit(request, grant_id):
         if not description_rich:
             description_rich = description
 
+        is_clr_eligible = json.loads(request.POST.get('is_clr_eligible', 'true'))
+        grant.is_clr_eligible = description_rich = is_clr_eligible
```
Comment (gdixon) [inline app/grants/views.py:1597]: Why are we assigning `description_rich = is_clr_eligible`, this will overwrite  `description_rich`?

Comment (thelostone-mc) [reply app/grants/views.py]: Ugh issues with copy pasting

**Review by gdixon — CHANGES_REQUESTED** [review]
Should we be only be setting this state when the grant is `active`? Otherwise users will be able to mark their grants as eligible before we've approved them?

File: app/grants/views.py (hunk 1 above)
Comment (gdixon) [reply app/grants/views.py]: Awesome! 😅

File: app/grants/views.py [hunk 2]
```diff
@@ -1593,6 +1593,9 @@ def grant_edit(request, grant_id):
         if not description_rich:
             description_rich = description
 
+        is_clr_eligible = json.loads(request.POST.get('is_clr_eligible', 'true'))
+        grant.is_clr_eligible = is_clr_eligible
```
Comment (gdixon) [inline app/grants/views.py:1597]: Can we wrap this with `if grant.active:`?

Comment (thelostone-mc) [reply app/grants/views.py:1597]: hmm well if a grant is not active -> this endpoint shouldn't be accessible via UI.
So ideally they would not be able to edit their grants right ?

Comment (thelostone-mc) [reply app/grants/views.py]: changed it anyways to be safer

**Pull request merged**